README.md

```diff
@@ -1,64 +1,82 @@
-## About
+## Open Source Routing Machine
 
-The Open Source Routing Machine is a high performance routing engine written in C++11 designed to run on OpenStreetMap data.
+[![Travis](https://travis-ci.org/Project-OSRM/osrm-backend.png?branch=master)](https://travis-ci.org/Project-OSRM/osrm-backend)
```

Comment:
I felt like the table was too clunky and putting the system names in front of the banners I was not able to align them properly with markdown. Should be fine with our other projects
- https://github.com/mapbox/mapbox-gl-js/blob/master/README.md
- https://github.com/Turfjs/turf/blob/master/README.md